src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -153,6 +151,12 @@ define(function (require, exports, module) {
             this.getFlux().actions.groups.setGroupExpansion(currentDocument, currentLayer,
                 !currentLayer.expanded, descendants);
         },
+        
+        _handleLayerNameInput: function (focused) {
```

Comment:
This requires JSDoc. I would also name it something more explicit about what it is doing. It's confusing, to me at least, to see a function name handleLayerNameInput which does nothing with the layer name being inputted.